docs/sources/reference/run.md

```diff
@@ -882,20 +882,27 @@ command is not available for this logging driver
 
 The following logging options are supported for this logging driver:
 
-    --log-opt address=[tcp|udp]://host:port
-    --log-opt address=unix://path
+    --log-opt syslog-address=[tcp|udp]://host:port
+    --log-opt syslog-address=unix://path
+    --log-opt syslog-tag="mailer"
 
-`address` specifies the remote syslog server address where the driver connects to.
+`syslog-address` specifies the remote syslog server address where the driver connects to.
```

Comment:
@LK4D4 this change was what made me think it was there before (but I may be confused)